### PR TITLE
add isolated ws

### PR DESCRIPTION
**Pull request by kilu01**
- add keep isolated alive key
- add isolated ws

### CODE REVIEW

File: lib/binance/margin/websocket/ws_isolated_client.ex [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      end
+
+      # Callbacks
+
+      def handle_pong(:pong, state) do
+        {:ok, inc_heartbeat(state)}
+      end
+
+      def handle_connect(_conn, state) do
+        :ok = info("Binance Margin Connected!")
```
Comment (sirius248) [inline lib/binance/margin/websocket/ws_isolated_client.ex]: Binance Isolated Margin Connected

File: lib/binance/margin/websocket/ws_isolated_client.ex [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+        {:ok, state}
+      end
+
+      def handle_info(
+            :keep_alive,
+            %{listen_key: listen_key, config: config, symbol: symbol} = state
+          ) do
+        Binance.Margin.keep_alive_isolated_listen_key(symbol, listen_key, config)
+        :ok = info("Keepalive Binance's User Data stream done!")
```
Comment (sirius248) [inline lib/binance/margin/websocket/ws_isolated_client.ex]: Binance Isolated Margin

File: lib/binance/margin/websocket/ws_isolated_client.ex (hunk 1 above)
Comment (kilu01) [reply lib/binance/margin/websocket/ws_isolated_client.ex]: Done

File: lib/binance/margin/websocket/ws_isolated_client.ex (hunk 2 above)
Comment (kilu01) [reply lib/binance/margin/websocket/ws_isolated_client.ex]: Done

**Review by sirius248 — APPROVED** [review]

**Pull request merged**